docs/general/adapters.md

```diff
@@ -0,0 +1,52 @@
+# Adapters
+
+AMS does this through two components: **serializers** and **adapters**.
+Serializers describe _which_ attributes and relationships should be serialized.
+Adapters describe _how_ attributes and relationships should be serialized.
+You can use one of the built-in adapters (```FlattenJSON``` is the default one) or create one by your one but you won't need to implement an adapter unless you wish to use a new format or
+media type with AMS.
+
+## Built in Adapters
+
+### FlattenJSON - Default
+
+It's the default adapter, it generates a json response without a root key.
+Doesn't follow any specifc convention.
+
+### JSON
+
+It also generates a json response but always with a root key. The root key **can't be overridden**, and will be automatically defined accordingly with the objects being serialized.
+Doesn't follow any specifc convention.
```

Comment:
I would argue that it can't be set from the controller or render command, but it can be overridden in the serializer or by subclassing the json adapter see https://github.com/rails-api/active_model_serializers/issues/986#issuecomment-119364320 though I'm not sure to what extent we want to highlight here, or link to it